requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 build~=0.7.0
+flake8
```

Comment:
Do we want to pin this to specific versions or no?

Reply:
i prefer not to pin. CI should complain if stuff breaks with the latest version, since that what most people will generally use.

if you pin here, you also need an upgrading/bumping process in place, and that's nonsensical for a low-activity project like this

Reply:
Fair enough, makes sense

Reply:
follow-up: #102